src/app/middleware/apm-agent.middleware.ts

```diff
@@ -9,11 +9,17 @@ class APMAgent {
     serviceVersion: config.app.version,
     serverUrl: config.elk.apm.host,
     captureHeaders: true,
+    captureBody: 'all',
+    usePathAsTransactionName: true,
   });
 
   public init() {
     return this._init.middleware.connect();
   }
+
+  public captureError(error) {
```

Comment:
I think no need to implement to capture method, it automatically logs the error
even tho if we dont log them

Reply:
It's not captured